Foundation/NSData.swift

```diff
@@ -439,6 +442,23 @@ open class NSData : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
 
     internal func makeTemporaryFile(inDirectory dirPath: String) throws -> (Int32, String) {
         let template = dirPath._nsObject.appendingPathComponent("tmp.XXXXXX")
+#if os(Windows)
+        let maxLength = Int(MAX_PATH) + 1
+        var buf: [UInt16] = Array<UInt16>(repeating: 0, count: maxLength)
+        let length = GetTempPathW(DWORD(MAX_PATH), &buf)
+        precondition(length <= MAX_PATH - 14, "temp path too long")
+        if GetTempFileNameW(buf, unsafeBitCast("SCF".utf16, to: LPCWSTR?.self),
+                            0, &buf) == FALSE {
```

Comment:
```suggestion
```

Reply:
Huh?  I don't understand the recommendation here.  This is part of the signature of `GetTempFileNameW`.

Reply:
Unfortunately you can’t do multi-line suggestions yet. That line got folded into my previous change.